spec/puppetserver/ca/local_certificate_authority_spec.rb

```diff
@@ -1,23 +1,50 @@
 require 'puppetserver/ca/local_certificate_authority'
+require 'puppetserver/ca/config/puppet'
 
+require 'utils/ssl'
 require 'puppetserver/ca/utils/signing_digest'
 
 RSpec.describe Puppetserver::Ca::LocalCertificateAuthority do
+  include Utils::SSL
 
   let(:settings) {
-    { :ca_ttl => 157680000,
-      :ca_name => 'bulla2',
-      :subject_alt_names => '',
-      :root_ca_name => 'bulla',
-      :certname => 'ulla',
-      :keylength => 512,
-      :hostprivkey => '$privatekeydir/$certname.pem',
-      :hostpubkey => '$publickeydir/$certname.pem',
-      :csr_attributes => '$confdir/csr_attributes.yaml',
-      :serial => '$cadir/serial' } }
+    with_ca_in(Dir.mktmpdir) do |config|
```

Comment:
Dir.mktmpdir without a block won't clean up the tmpdir it creates, it will though if we use a block, though I don't think returning the config will work quite the same way.

Reply:
I've changed the tests to delete the tmp dir in an `after` block.